src/Nordea/Components/Card.elm

```diff
@@ -47,9 +47,7 @@ view : List (Attribute msg) -> List (Html msg) -> Card msg -> Html msg
 view attrs children (Card config) =
     let
         cardStyle =
-            [ displayFlex
-            , flexDirection column
-            , borderRadius (rem 0.5)
+            [ borderRadius (rem 0.5)
```

Comment:
Hmm, I suggest we remove the `cardContentContainer` instead as that would make it easier to use. 

Adding padding to this, and `marginBottom (rem 1)` to the `cardTitle` (instead of the padding) does the trick I think
```suggestion
            [ displayFlex
            , flexDirection column
            , borderRadius (rem 0.5)
            , padding (rem 2)
```